packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -164,7 +159,53 @@ class DriveCommand extends RunCommandBase {
         );
       }
 
-      final LaunchResult result = await appStarter(this);
+      if (isWebPlatform && getBuildInfo().isDebug) {
+        // TODO(angjieli): remove this once running against
+        // target under test_driver in debug mode is supported
+        throwToolExit(
+          'Flutter Driver web does not support running in debug mode.\n'
+          '\n'
+          'Use --profile mode for testing application performance.\n'
+          'Use --release mode for testing correctness (with assertions).'
```

Comment:
Are we going to address this in the next PRs?

As a team we are also have usecase for the driver tests in the debug mode as well. Thanks!

Reply:
This is not a driver bug so @jonahwilliams probably knows more about it.
Currently, you will also get an error when running `flutter run --target=test_driver/scroll_perf_web.dart --debug`.

Reply:
Actually, this should probably work at ToT